app-misc/kas/kas-4.5.ebuild

```
@@ -0,0 +1,32 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{6..13} )
```

< 3.10 is a noop in Gentoo:
```suggestion
PYTHON_COMPAT=( python3_{10..13} )
```